StPetersburg/InEtn40558.xml

```diff
@@ -0,0 +1,251 @@
+<?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="InEtn40558" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Collection of Magic Texts</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DE"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens: Eine multimediale
+                            Forschungsumgebung / beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+              <msDesc type="scroll">
```

Comment:
I don't think that this  encoding is mentioned in our guidelines, and there is already  `<objectDesc form="Scroll">`, so I see no need for it. But I see now that you have used this in all records. Maybe I missed a discussion, sorry. Was there a particular motivation behind having the information here?

Reply:
Actually I can not rememeber either, how this appeared in my encoding. I suppose that I have made a mistake. This should rather be: xml:id="ms", right?

Reply:
Yes, at least this is what we have everywhere. I never thought about it :)

Reply:
I have corrected this in this PR, I will go through my other records and correct it as well